flag.go

```diff
@@ -1005,7 +1007,11 @@ func (f *FlagSet) parseArgs(args []string, fn parseFunc) (err error) {
 			args, err = f.parseShortArg(s, args, fn)
 		}
 		if err != nil {
-			return
+			if f.errorHandling != IgnoreOnError {
```

Comment:
Yeah, I wanted that in the first place too... but it has to be checked on every individual arg, and `Parse()` only knows the accumulated result.